gen/definitions/bgp_peer_local_asn.yaml

```diff
@@ -0,0 +1,78 @@
+---
+name: BGP Peer Local ASN
+class_name: bgpLocalAsn
+dn: sys/bgp/inst/dom-[%s]/peer-[%s]/localasn
+ds_description: This data source can read the BGP peer local asn configuration.
+res_description: This resource can manage the BGP peer local asn configuration.
+doc_path: Routing%20and%20Forwarding/bgp:localasn/
+doc_category: BGP
+parents:
+  - BGP Peer
```

Comment:
We also need to add this new resource as a child under the "BGP Peer" resource.

Reply:
Done